local-tests/test_major_sync.py

```diff
@@ -0,0 +1,66 @@
+#!/bin/env python
+import os
+import sys
+from os.path import abspath, join
+from time import sleep, ctime
+
+from chainrunner import Chain, Seq, generate_keys, check_finalized
+
+
+def printt(s): print(ctime() + ' | ' + s)
+
+
+workdir = abspath(os.getenv('WORKDIR', '/tmp/workdir'))
+binary = abspath(os.getenv('ALEPH_NODE_BINARY', join(workdir, 'aleph-node')))
+
+phrases = [f'//{i}' for i in range(5)]
+keys = generate_keys(binary, phrases)
+all_accounts = list(keys.values())
+chain = Chain(workdir)
+printt('Bootstraping the chain with binary')
+chain.bootstrap(binary,
+                all_accounts,
+                nonvalidators=None,
+                sudo_account_id=keys[phrases[0]],
+                chain_type='local')
+
+chain.set_flags('no-mdns',
+                port=Seq(30334),
+                validator_port=Seq(30343),
+                ws_port=Seq(9944),
+                rpc_port=Seq(9933),
+                unit_creation_delay=200,
+                execution='Native')
+addresses = [n.address() for n in chain]
+validator_addresses = [n.validator_address() for n in chain]
+chain.set_flags(bootnodes=addresses[0])
+chain.set_flags_validator(public_addr=addresses,
+                          public_validator_addresses=validator_addresses)
+
+chain.set_flags_validator('validator')
+
+printt('Starting the chain')
+chain.start('aleph', nodes=[0, 1, 2, 3])
+
+# sleep 2h to produce around 5/6 * 7200 blocks > 3 * 1800 (max forest depth).
```

Comment:
Hm, it would likely be better to have a DB snapshot with such a state already set up, and here at most check whether it works at all (produce a small amount of blocks and check finalization). Not a blocker for this PR, but something we should consider talking with infra about. Could you make a task for that? Unless you think you can organize it quickly ofc.

Reply:
it would be better, w8 for @Marcin-Radecki input before creating a ticket. there is also something like export/import blocks commands in cli that could be helpful here

Reply:
Unfortunately, this is a blocker to this PR as the amount of GH-free minutes would be spent after 17 days (2000 minutes / 120). 
@MikolajGasior  can we spin a tiny self-hosted machine for this test? What would be its cost (PM us please)

Reply:
If this is enough of a problem then I would definitely suggest solving it properly, i.e. generating a snapshot (once/rarely?) and only downloading it in the test. If the snapshot is from ~2h then this would download almost immediately, thus likely removing any problem with GitHub minutes.

Reply:
Ok done, I have zipped workdir with 6k > blocks in db plus this one 'late' node starts from zero